assets/xml/objects/object_mu.xml

```diff
@@ -1,62 +1,63 @@
+        <Texture Name="gHoneyAndDarlingWomanDressTex" OutName="honey_and_darling_woman_dress" Format="ci8" Width="16" Height="16" Offset="0x56F0" />
+        <Texture Name="gHoneyAndDarlingJointShadeTex" OutName="honey_and_darling_joint_shade" Format="i8" Width="8" Height="8" Offset="0x57F0" /> <!-- For the man's shoulders, elbows, and knees, and the woman's elbows -->
+        <Texture Name="gHoneyAndDarlingHeadSkinTex" OutName="honey_and_darling_head_skin" Format="ci8" Width="8" Height="8" Offset="0x5830" /> <!-- For their necks, under chin, noses, cheeks, and foreheads -->
+        <Texture Name="gHoneyAndDarlingManHairTex" OutName="honey_and_darling_man_hair" Format="ci8" Width="16" Height="16" Offset="0x5870" /> 
+        <Texture Name="gHoneyAndDarlingEarTex" OutName="honey_and_darling_ear" Format="ci8" Width="8" Height="8" Offset="0x5970" />
+        <Texture Name="gHoneyAndDarlingMouthTex" OutName="honey_and_darling_mouth" Format="ci8" Width="32" Height="16" Offset="0x59B0" />
+        <Texture Name="gHoneyAndDarlingManEyeTex" OutName="honey_and_darling_man_eye" Format="ci8" Width="32" Height="32" Offset="0x5BB0" />
+        <Texture Name="gHoneyAndDarlingHandTex" OutName="honey_and_darling_hand" Format="ci8" Width="8" Height="8" Offset="0x5FB0" />
+        <Texture Name="gHoneyAndDarlingWomanHairTex" OutName="honey_and_darling_woman_hair" Format="ci8" Width="8" Height="16" Offset="0x5FF0" />
+        <Texture Name="gHoneyAndDarlingWomanEyeTex" OutName="honey_and_darling_woman_eye" Format="ci8" Width="32" Height="32" Offset="0x6070" />
+        <DList Name="gHoneyAndDarlingLegsDL" Offset="0x9370" />
+        <DList Name="gHoneyAndDarlingWomanHeadDL" Offset="0x96C8" />
+        <DList Name="gHoneyAndDarlingWomanRightHandDL" Offset="0x9BC8" />
+        <DList Name="gHoneyAndDarlingWomanRightLowerArmDL" Offset="0x9D10" />
+        <DList Name="gHoneyAndDarlingWomanRightUpperArmDL" Offset="0x9E10" />
+        <DList Name="gHoneyAndDarlingWomanLeftHandDL" Offset="0x9F10" />
+        <DList Name="gHoneyAndDarlingWomanLeftLowerArmDL" Offset="0xA058" />
+        <DList Name="gHoneyAndDarlingWomanLeftUpperArmDL" Offset="0xA158" />
+        <DList Name="gHoneyAndDarlingWomanTorsoDL" Offset="0xA258" />
+        <DList Name="gHoneyAndDarlingManTorsoDL" Offset="0xA3C8" />
+        <DList Name="gHoneyAndDarlingManHeadDL" Offset="0xA510" />
+        <DList Name="gHoneyAndDarlingManRightUpperArmDL" Offset="0xAA10" />
+        <DList Name="gHoneyAndDarlingManRightLowerArmDL" Offset="0xAB10" />
+        <DList Name="gHoneyAndDarlingManRightHandDL" Offset="0xAC10" />
+        <DList Name="gHoneyAndDarlingManLeftUpperArmDL" Offset="0xAD58" />
+        <DList Name="gHoneyAndDarlingManLeftLowerArmDL" Offset="0xAE58" />
+        <DList Name="gHoneyAndDarlingManLeftHandDL" Offset="0xAF58" />
+        <DList Name="object_mu_DL_00B0A0" Offset="0xB0A0" /> <!-- TODO: haven't been able to render these -->
         <DList Name="object_mu_DL_00B0E0" Offset="0xB0E0" />
```

Comment:
I had trouble getting these dlists to render anything besides a black square in Z64Utils... they seem related to the heart texture? Any tips are welcome, or guidance on what to do, here.

Reply:
These are best going to be named with context, as they work in conjunction with each other so are hard to determine by themselves.

`object_mu_DL_00B0A0` could be named `gHoneyAndDarlingManHeartMaterialDL`. This DL is used to setup the materials of the DL like colors, draw modes, and anything else it needs. 
`object_mu_DL_00B0E0` could be named `gHoneyAndDarlingManHeartModelDL`. This DL holds the actual model to be drawn.
If you have questions or would like more of an explanation feel free to ask in the Discord, I don't want to flood github with a long explanation.  You can also search for `MaterialDL` and `ModelDL` throughout the codebase for examples.

Also feel free to name `gameplay_keep_Tex_05E6F0`. Just remember to reserve that particular texture on the `Gameplay_keep` tab of the objects sheet.

Reply:
Thanks for the help! I did find a couple other examples and it makes more sense, now.

Also renamed `gameplay_keep_Tex_05E6F0` to `gEffHeartTex`